operator/pkg/reconciliation/reconcile_racks.go

```diff
@@ -573,11 +573,6 @@ func (rc *ReconciliationContext) CheckPodsReady(endpointData httphelper.CassMeta
 	desiredSize := int(rc.Datacenter.Spec.Size)
 
 	if desiredSize == readyPodCount && desiredSize == startedLabelCount {
-		// When the ready and started counts match the desired counts and nodeStarted is true, then that means we have
-		// just started the last node in the data center.
-		if nodeStarted {
-			return result.RequeueSoon(2)
-		}
```

Comment:
This is both probably not necessary due to earlier bug fixes and not sufficiently explicit. We are completely dependent on the comment here to understand what this is about. This has been replaced with `CheckCassandraNodeStatuses()` which makes explicit that we must collect all Host ID information.